otter/cloud_client.py

```diff
@@ -345,13 +347,14 @@ def _log_it(result):
         # So we can link it to any non-cloud_client logs
         request_id = resp.request.headers.getRawHeaders(
             'x-otter-request-id', [None])[0]
-
+        resp_body = (
+            json.dumps(response_body_filter(json_body), sort_keys=True)
```

Comment:
I am guessing due to ES mapping issues. This way it always remains string and doesn't confuse ES indexes if for some reason different kind of values are returned.